src/helpers/sorts.ts

```diff
@@ -47,6 +48,14 @@ function getAllMeasuresSorts(afm: AFM.IAfm): AFM.SortItem[] {
     }, []);
 }
 
+export function getDefaultBarChartSort(
```

Comment:
this function will not be necesery after changes suggested by Lan

Reply:
This is replaced with `getDefaultBarChartSort` from `src/internal/utils/sort.ts`